error_internal.go

```diff
@@ -0,0 +1,12 @@
+package helmclient
+
+import "strings"
+
+// isNoHostError asserts no route to Host error.
```

Comment:
```suggestion
// isNoSuchHostError asserts no such host error.
```

The error message text "no such host" is descriptive for me. So let's align our code with that.